tests/fe/fe_enriched_color_08.cc

```diff
@@ -45,9 +45,6 @@
 
 #include "../tests.h"
 
-#undef DEBUG
```

Comment:
This test of course totally subverts the idea of `DEBUG`. I removed this `#undef` and where it affected the code (in line 283 below), just commented out the code that previously was guarded by the disabled `#ifdef DEBUG`.